vault/base/configmaps/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+data:
+  extraconfig-from-values.hcl: |-
+    disable_mlock = true
+    ui = true
+    listener "tcp" {
+      tls_disable = 1
+      address = "[::]:8200"
+      cluster_address = "[::]:8201"
+    }
+    storage "raft" {
+      path = "/vault/data"
+    }
+    service_registration "kubernetes" {}
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/instance: opf
```

Comment:
Some of these resources have `app.kubernetes.io/instance: opf` and others have `app.kubernetes.io/instance: nerc`. We should probably just remove these from the individual manifests and apply them via the top-level `kustomization.yaml`.

If not that, we should at least ensure that they are all consistent.